### ISSUE TITLE
timescaledb-docker-ha VS timescaledb-docker

### ISSUE THREAD

**Issue by shusaan** [timescale/timescaledb-docker#250]
HI I have query about "[timescaledb-docker-ha](https://github.com/timescale/timescaledb-docker-ha)" and "[timescaledb-docker](https://github.com/timescale/timescaledb-docker)" container image.
timescaledb-docker-ha is based ubuntu & install extra packages and extensions as well, on the other hand timescaledb-docker is based on alpine and only install timescaledb-* related packages, right?
I want to run "[timescaledb-docker](https://github.com/timescale/timescaledb-docker)" image using [CNPG](https://cloudnative-pg.io/), is this image is ready for production use?
because we want to run this image in production due to lightweight and most important it has less vulnerabilities.

**Comment by antekresic** [comment]
Yes, it is production ready, it just doesn't include some extra packages and and extensions which are in the HA image.

Hope this helps.

**Comment by einarwar** [comment]
Chiming in here since it is partially related. According to the timescale docs: 

> The [TimescaleDB HA](https://hub.docker.com/r/timescale/timescaledb-ha) Docker image offers the most complete TimescaleDB experience. It uses [Ubuntu](https://ubuntu.com/), includes [TimescaleDB Toolkit](https://github.com/timescale/timescaledb-toolkit), and support for PostGIS and Patroni. The lighter-weight timescale/timescaledb:latest-pg17 non-ha image uses [Alpine](https://alpinelinux.org/).

When inspecting the images for e.g amd64 however, the [timescaledb:latest-pg17](https://hub.docker.com/layers/timescale/timescaledb/latest-pg17/images/sha256-f2c984dedaa9bf4c464ab3ecb1a1a88a764f03b3686ac453f2e7744c1d33fdc7) is 637 MB, while the [tiemscaledb-ha:pg17](https://hub.docker.com/layers/timescale/timescaledb-ha/pg17/images/sha256-50338116a19bafc795ffe28e1cd243fcfab6335d843c1f3504642759a529793b) is 566 MB, which seems to contradict the docs? Could you give any clarity here @antekresic ?

**Comment by fabriziomello** [comment]
@einarwar maybe you're looking to the wrong images, the correct ones are:

[timescaledb:latest-pg17](https://hub.docker.com/layers/timescale/timescaledb/latest-pg17/images/sha256-7c89211ba7680ebef54b2ff74418ead7a91d81e4bf8ebc703170cb62c7b0cc11)
[timescale/timescaledb-ha:pg17.4-ts2.19.3-all](https://hub.docker.com/layers/timescale/timescaledb-ha/pg17.4-ts2.19.3-all/images/sha256-023f73fecc7d714f72bd493f2e20611fef59c52051081d1193cdef9caaf2194e)
